Formula/hyx.rb

```diff
@@ -0,0 +1,23 @@
+class Hyx < Formula
+  desc "Powerful hex editor for the console"
+  homepage "https://yx7.cc/code/"
+  url "https://yx7.cc/code/hyx/hyx-2021.06.09.tar.xz"
+  sha256 "8d4f14e58584d6cc8f04e43ca38042eed218882a389249c20b086730256da5eb"
+  license "MIT"
+
+  uses_from_macos "expect" => :test
+
+  def install
+    ENV["CFLAGS"] = "-Os"
```

Comment:
Why is this one needed?

Reply:
The Makefile defines:

```make
CFLAGS ?= -O2 -Wl,-s \
               -Wl,-z,relro,-z,now -fpic -pie -D_FORTIFY_SOURCE=2 -fstack-protector-all
```
The `-Wl,-s -Wl,-z,relro,-z,now -fpic -pie` doesn't work on macOS - as *something* needs to be set here, if it's preferrable I set CFLAGS to `-O2 -D_FORTIFY_SOURCE=2 -fstack-protector-all` instead.

Already reported upstream.

Reply:
Fixed in https://github.com/Homebrew/homebrew-core/pull/106085/commits/7ad9e18b8c235840453710d7a6e45a21ce394373 - CFLAGS now matches upstream, except those which are incompatible.